remappings.txt

```diff
@@ -2,6 +2,7 @@ ds-test/=node_modules/ds-test/src
 forge-std/=node_modules/forge-std/src
 forge-gas-snapshot/=node_modules/forge-gas-snapshot/src
 solmate/=node_modules/solmate/src
+@cowprotocol/=node_modules/@cowprotocol/contracts/src/contracts
 cowprotocol/=node_modules/@cowprotocol/contracts/src/
 composable-cow/=node_modules/composable-cow/
```

Comment:
comment, wdyt about using `@composable-cow` and adding a comment on why are we declaring the `cowprotocol` remapping, that is internal to their codebase